package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "axios": "^0.19.0",
     "bootstrap": "^4.3.1",
+    "bootstrap-vue": "^2.0.0-rc.27",
```

Comment:
I had to remove this dependency in the previous versions as i wanted to keep the number of external package dependencies to really minimum. 
Can you use the native bootstrap provided markup to render the elements like its done in the other places ?